client/state/data-layer/wpcom/sites/admin-menu/test/index.js

```diff
@@ -60,6 +58,22 @@ describe( 'handlers', () => {
 					},
 				],
 			},
+			{
+				icon: 'dashicons-default',
+				slug: 'my-custom-menu-3',
+				title: 'Home',
+				type: 'menu-item',
+				url: '/home',
+				children: [
+					{
+						parent: 'my-custom-menu-3',
+						slug: 'my-custom-menu-4',
+						title: 'Settings',
+						type: 'submenu-item',
+						url: 'https://example.wordpress.com/wp-admin/settings.php',
+					},
+				],
+			},
```

Comment:
```suggestion
			{
				icon: 'dashicons-default',
				slug: 'my-custom-menu-3',
				title: 'Home',
				type: 'menu-item',
				url: '/home',
				children: [
					{
						parent: 'my-custom-menu-3',
						slug: 'my-custom-menu-3',
						title: 'Settings',
						type: 'submenu-item',
						url: 'https://example.wordpress.com/wp-admin/settings.php',
					},
                    {
						parent: 'my-custom-menu-4',
						slug: 'my-custom-menu-4',
						title: 'Malicious Child',
						type: 'submenu-item',
						url: 'javascript:alert("not good")',
					},
				],
			},
```

As we're testing we're santising the `children` for malicious content could we add another evil JS script injection as a child item?